qemu/tests/cfg/nvdimm.cfg

```diff
@@ -39,6 +39,7 @@
     share_mem = yes
     nv_file = "${mount_dir}/nv0"
     depends_pkgs = "ndctl "
+    repo_install_cmd = "dnf install -y https://kojipkgs.fedoraproject.org//packages/pandoc/3.0.1/24.fc39/x86_64/pandoc-3.0.1-24.fc39.x86_64.rpm https://kojipkgs.fedoraproject.org//packages/pandoc/3.0.1/24.fc39/noarch/pandoc-common-3.0.1-24.fc39.noarch.rpm"
```

Comment:
Hello @mcasquer, In my perception, pandoc is used to convert doc files. Can you remove pandoc from the dep list and test again, I don't think we need to cover doc test.

Reply:
@PaulYuuu `pandoc` package is mandatory for compiling the nvml/pmdk project:
```
[root@localhost nvml]# make
test -f .skip-doc || make -C doc all
make[1]: Entering directory '/home/nvml/doc'
../utils/md2man.sh libpmem/libpmem.7.md default.man libpmem/libpmem.7
../utils/md2man.sh: line 50: pandoc: command not found
make[1]: *** [Makefile:198: libpmem/libpmem.7] Error 127
make[1]: Leaving directory '/home/nvml/doc'
make: *** [Makefile:57: doc] Error 2
```

Reply:
Refer to the source code, you can use `make DOC=n check` to skip make doc and check-doc.
`make check` will call `make all`, so the first `make` is not need, just `make DOC=n check` for nvml(AKA pmdk, the project renamed).

Reply:
@PaulYuuu good catch! I'll update the test to skip the documentation, thanks !

Reply:
@PaulYuuu I see now there are a lot of warning messages for the `make DOC=n check` but this can be handled in a different patch

Reply:
OK, I checked that. the `check` target should not include the `check-doc` target when `DOC=n`. To bypass this I think you can refer to https://github.com/pmem/pmdk/blob/master/src/test/README.
```bash
make DOC=n
cd src/test
cp testconfig.sh.example testconfig.sh
./RUNTESTS.sh
```
There's also a RUNTESTS.py for it if you think it's worth testing.